app/controllers/application_controller.rb

```diff
@@ -34,11 +34,14 @@ def current_website
 
   def set_website(website)
     return unless logged_in?
-    @website = website
-    session[:epiclogger_website_id] = @website.id
+    unless website.nil?
```

Comment:
use `return unless website.nil?` one line above the 2 that were below. So we can keep a clean look to the code.